conf/mjolnir_globe

```diff
@@ -0,0 +1,25 @@
+//Profile config names for nf-core/configs
+params {
+  config_profile_description = 'Section for Hologenomics @ Globe Institute, University of Copenhagen - mjolnir_globe profile provided by nf-core/configs.'
+  config_profile_contact = 'Aashild Vaagene (@ashildv)'
+  config_profile_url = 'https://globe.ku.dk/research/hologenomics/'
+  max_memory = 500.GB
+  max_cpus = 50
+  max_time = 720.h
+}
+
+singularity {
+        enabled = true
+        autoMounts = true
+        cacheDir = '/maps/projects/mjolnir1/data/cache/nf-eager/singularity'
```

Comment:
cacheDir is also very eager specific

Reply:
Thanks for spotting that, a new cacheDir was created and I have updated the path